src/components/PostPageDetail/index.js

```diff
@@ -14,6 +14,7 @@ import {
 import Toast from 'react-native-simple-toast';
 
 import {Footer} from '..';
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement on this line seems to be redundant as there are no new imports added. It might have been left unintentionally during the code changes.
```diff
- import {Footer} from '..';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->